### DISCUSSION TITLE
Problem Cortex Analyzer Integration

### DISCUSSION THREAD

**Discussion by DaliBd** [Security-Onion-Solutions/securityonion#5571] [Unsupported Versions]
I need to integrate Cortex Analyzer with Security Onion 2.3 but always not working.
I followed the Cortex documentations, but when i run this command i have a lot of errors by pip2

![image](https://user-images.githubusercontent.com/90970037/133890294-38e376b4-de39-4f34-bbfd-cd0f2cade331.png)

Output : 

![image](https://user-images.githubusercontent.com/90970037/133890320-b2fd3bb5-ba6e-4446-be32-586621000b4d.png)

**Comment by weslambert** [comment]
Are you trying to add a custom analyzer, or configure an existing analyzer?
